taipy/core/taipy.py

```diff
@@ -368,6 +354,8 @@ def delete(entity_id: Union[TaskId, DataNodeId, SequenceId, ScenarioId, JobId, C
       the scenario can be deleted.
```

Comment:
yep it does, `_get_children_entity_ids` in `taipy/core/scenario/_scenario_manager.py` also contains finding the `Submission` under the scenario so it can be deleted along with it as well